### PR TITLE
Add section on "floating" agents

### LINKED ISSUE #119: Describe how to move off-host agents automatically
Several users (especially those who are using RDS) would like to move agents transparently between servers.

1. Create the file /etc/vividcortex/vc-agent-007.conf with the following:
    ```

    {
        "hostname": {{ some unique hostname string }}
    }
    ```
2. Install VividCortex on the server as usual
3. Add your RDS instances using the UI

When moving agents to another server...

1. Shutdown agents on the original server
2. Setup /etc/vividcortex/vc-agent-007.conf *with the same hostname* as before
3. Install VividCortex

### PR DESCRIPTION
**Pull request by Preetam**
Closes #119.

Ping @gkristic or @potocnyj

### CODE REVIEW

File: content/getting-started/installing.md [hunk 1]
```diff
@@ -69,6 +69,32 @@ operating-system host of your choice.
 
 Note that there is no off-host support for monitoring queries for MongoDB or Redis, because they do not expose running queries the way MySQL or PostgreSQL do.
 
+### Off-host monitoring with "floating" agents
```
Comment (potocnyj) [inline content/getting-started/installing.md]: I think this more naturally belongs under the Managing Hosts page than under Installing

Comment (Preetam) [inline content/getting-started/installing.md]: Yeah, I agree. I didn't know that was a section.

File: content/getting-started/installing.md [hunk 2]
```diff
@@ -69,6 +69,32 @@ operating-system host of your choice.
...
+
+`{{ unique_os_hostname }}` should be replaced with a unique identifier.
+The hostname does not have to be the actual hostname of the server. You just need
+to use the same hostname when moving agents to new servers. For example, you could
+use `rds-monitor-1`. Next, install VividCortex on the server as usual, and set up
+off-host monitoring through the UI.
+
+To move agents to another server, shutdown agents, ensure that the `vc-agent-007.conf`
+file has the same hostname on the new server, and install VividCortex on the new
+server. The off-host agent and credential configuration will follow to the new server.
```
Comment (potocnyj) [inline content/getting-started/installing.md]: I'd rephrase this a little to reflect that the host in the UI will not change at all (apart from the restart event for the agents). Other than that though, there should be no visible side-effects from the migration.

File: content/getting-started/installing.md (hunk 1 above)
Comment (potocnyj) [inline content/getting-started/installing.md]: It might be worth cross linking from here after you move it, otherwise users probably won't see the instructions until they want to move a host without side-effects 😄
